Makefile

```diff
@@ -14,7 +14,7 @@ test:
 .PHONY: test
 
 unit:
-	go test -v .
+	go test -v -p 1 ./...
```

Comment:
Do we need to specify the `-p` flag here? We don't do it normally

Reply:
I noticed occasional unit flakes, which were part of previous test code, due to attempts to operate on a single ClusterOperator resource in parallel, and not allowing it to clean results from previous execution.

Reply:
Is the default not to run tests serially anyway? I had thought this change isn't actually doing anything when I first read it

Reply:
Still not sure why this makes a difference, have we declared explicitly that some of our tests are parallel?

Reply:
You are right, they are. I just checked. We will probably just need something which ensures the resource was actually deleted. Depends on the frequency of the flakes.

Reply:
Let's remove this for now and fix in a separate PR